sql/core/src/test/scala/org/apache/spark/sql/DatasetBenchmark.scala

```diff
@@ -242,75 +248,20 @@ object DatasetBenchmark {
     benchmark
   }
 
-  def main(args: Array[String]): Unit = {
-    val spark = SparkSession.builder
-      .master("local[*]")
-      .appName("Dataset benchmark")
-      .getOrCreate()
+  val spark = SparkSession.builder
+    .master("local[*]")
+    .appName("Dataset benchmark")
+    .getOrCreate()
```

Comment:
Can we move this SparkSession building part into `benchmark()` function and before `runBenchmark("Dataset Benchmark")`?